src/assets.js

```diff
@@ -647,19 +647,24 @@ class Assets extends Events {
      * @param {object} options - Options
      * @param {string} options.name - The asset name
      * @param {object} options.json - The JSON
+     * @param {number} options.spaces - The number of spaces used for indentation. Defaults to 0
+     * (tightly packed output).
      * @param {Asset} options.folder - The parent folder asset
      * @param {boolean} options.preload - Whether to preload the asset. Defaults to true.
      * @param {Function} options.onProgress - Function to report progress
      * @returns {Promise<Asset>} The new asset
      */
     createJson(options = {}) {
+        const spaces = options.spaces !== undefined ? options.spaces : 0;
```

Comment:
```suggestion
        const spaces = options.spaces ?? 0;
```

Reply:
Cool, I didn't know about that operator! ❤️